### PR TITLE
cleaned up a couple of misspellings...

### PR DESCRIPTION
**Pull request by bcarancibia**
Everything looks pretty good to me, found a couple of misspellings.

The only thing I was unsure of is in the code there is a covriates variable, but is consistent throughout so I didn't know if that was on purpose or not.

Example:

```
covriates <- all.vars(model)
```

### CODE REVIEW

File: R/ps.R [hunk 1]
```diff
@@ -139,10 +139,10 @@ print.prop_scr <- function(x, ..., n = 10){
   cli_h1("Model")
   cli_bullets(c("*" = f_rhs(x$model)))
 
-  cli_h1("Propensoity Scores and Weights")
+  cli_h1("Propensity Scores and Weights")
   x$external_df |>
     select(!!x$id_col,
-           Internal = .data$`___internal___`,
+           `Internal` = .data$`___internal___`,
```
Comment (statasaurus) [inline R/ps.R:145]: This doesn't need back ticks. It is just for printing. The others have back ticks cause of the spaces

**Review by statasaurus — CHANGES_REQUESTED** [review]
Looks good I would just remove the backticks

File: R/ps.R (hunk 1 above)
Comment (bcarancibia) [reply R/ps.R]: fixed

**Review by statasaurus — APPROVED** [review]

**Comment by bcarancibia** [comment]
all done! closed

**Pull request merged**